bodhi/client/bindings.py

```diff
@@ -725,6 +738,20 @@ def update_str(self, update, minimal=False):
                 update['autokarma'], update['unstable_karma'], update['stable_karma']))
         ]
 
+        try:
+            test_status = self.get_test_status(update['alias'])
+        except (ServerError, requests.exceptions.RequestException):
+            test_status = None
+
+        if test_status:
+            info = None
+            if 'errors' in test_status:
+                info = '\n'.join([el.description for el in test_status.errors])
+            elif 'decision' in test_status:
```

Comment:
Can we have any other case here ? Or could we do something like :  
``` 
if 'decision' in test_status: 
    info = .... (we have the decision)
else:
    info = .... (this is an error)
```

Reply:
I had this construct first but that lead to errors in the unit-tests. A bunch of the test define what ``send_request`` returns so that it returns the JSON representation of an update (for example) and that would make this code fail since it couldn't find neither ``errors`` nor ``decision``, so I went with this construct albeit a little more odd but leading to less changes overall.

Reply:
Ok let's keep the current logic then :+1: